docs/projects/tad/reporting-standard/latest.md

```diff
@@ -113,10 +113,25 @@ legal base the following fields are present.
     2. `link` (OPTIONAL, string). URI pointing towards the contents of the law.
 17. `used_data` (OPTIONAL, string). An overview of the data that is used in the system.
 18. `technical_design` (OPTIONAL, string). Description on how the system works.
-19. `external_providers` (OPTIONAL, list[string]). Name of an external provider, if relevant. There can
-be multiple external providers.
+19. `external_providers` (OPTIONAL, list). If relevant, these fields allow to store information on external providers.
+ There can be multiple external providers.
+    1. `name` (OPTIONAL, string). Name of the external provider.
+    2. `version` (OPTIONAL, string). Version of the external provider reflecting its relation to previous versions.
 20. `references` (OPTIONAL, list[string]). Additional reference URI's that point information about the system
 and are relevant.
+21. `interaction_details` (OPTIONAL, list[string]). Explain how the AI system interacts with hardware or software,
+including other AI systems, or how the AI system can be used to interact with hardware or software.
+22. `version_requirements` (OPTIONAL, string). Describe the versions of the relevant software or firmware,
+ and any requirements related to version updates.
+23. `deployment_variants` (OPTIONAL, string). Description of all the forms in which the AI system is placed
+on the market or put into service, such as software packages embedded into hardware, downloads, or APIs.
+24. `hardware_requirements` (OPTIONAL, string). Provide a description of the hardware on which the AI system must be run.
+25. `product_markings` (OPTIONAL, string). If the AI system is a component of products, photos, or illustrations,
+describe the external features, markings, and internal layout of those products.
```

Comment:
Are these items indeed a string, or would a list be better?

Reply:
Yes indeed, I think so too!